reporting-anything/src/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "reporting-anything",
+  "description": "Download reports from anywhere",
+  "main": "download_report.js",
+  "type": "module",
+  "dependencies": {
+    "argparse": "^2.0.1",
+    "dompurify": "^2.1.1",
+    "fs": "^0.0.1-security",
```

Comment:
what is this?

Reply:
fs is used to download the file- this was the default version installed when I ran `yarn add fs`

Reply:
i think you can remove it, this seems to be an empty placeholder. fs should come by default